distribution/docs/pom.xml

```diff
@@ -45,308 +45,285 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-    </build>
-    <profiles>
-        <profile>
-            <id>documentation</id>
-            <activation>
-                <property>
-                    <name>skipDocs</name>
-                    <value>!true</value>
-                </property>
-            </activation>
-            <properties>
-                <packaging>pom</packaging>
-                <module>docs</module>
-                <skipTests>false</skipTests>
-            </properties>
-            <dependencies>
-                <dependency>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-dependency-plugin</artifactId>
+                    <version>2.10</version>
+                    <executions>
+                        <execution>
+                            <id>unpack</id>
+                            <phase>validate</phase>
```

Comment:
❓ For the `unpack`, `copy-adocs` and `copy-images`, would it make more sense to use the `generate-resources` or `process-resources` phase instead?